### ISSUE TITLE
Reference to UOM definition

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ets-gmljpx20#32]
Requirement 8 in document GMLJP2 v2.0 (OGC 08-085r4) only applies "In those cases where a UoM is identified by reference to an authority and code".

This means that tests should not fail because of not using URIs. 

Note that in GMLJP2 v2.1, many of the examples do not use URIs.

![Screenshot 2020-02-21 at 11 07 16](https://user-images.githubusercontent.com/18121038/75030244-9bdf7b00-549b-11ea-9185-bcb9a10cd40e.png)

**Comment by ghobona** [comment]
This can be fixed by changing the lines starting 438 in [org.opengis.cite.gmljpx20.core.CoreTests](https://github.com/opengeospatial/ets-gmljpx20/blob/master/src/main/java/org/opengis/cite/gmljpx20/core/CoreTests.java#L438) as shown below.

```java
            for(int i = 0; i < uomElements.getLength(); i++) {
            	Node element = uomElements.item(i);     
            	
            	 String uomCode = element.getAttributes().getNamedItem("code").getTextContent();
            	 
            	 boolean hasHttpUom = ( uomCode.startsWith("http"));  
            	 
            	 boolean hasAllowableUomNamespace = (uomCode.startsWith("http://www.opengis.net/def/uom/") || 
        				 uomCode.startsWith("https://www.opengis.net/def/uom/") || 
        				 uomCode.startsWith("http://www.epsg-registry.org"));
            	 
            	 if (hasHttpUom) {
            		 if(!hasAllowableUomNamespace) {
            			 
                             throw new AssertionError( ErrorMessage.get( GMLJP2_GMLCOV_UOM_HTTP ) );
            		 }
                 }
            }

```

**Comment by keshavnangare** [comment]
Added fix in PR #30.
